src/export/StructureDefinitionExporter.ts

```diff
@@ -158,17 +158,12 @@ export class StructureDefinitionExporter {
       // Maybe it's a FSH-defined definition and not a FHIR one
     } else {
       let structDef = cloneDeep(
-        this.structDefs.find(sd => sd.name === type || sd.id === type || sd.url === type)
+        [...this.profileDefs, ...this.extensionDefs].find(
```

Comment:
This isn't a breaking issue or anything, but I wonder if you think it would be reasonable to still have `this.structDefs` be a getter that returns `[...this.profileDefs, ... this.extensionDefs]`. It seems to be used in a couple places, so it might be useful to have.

Reply:
Yeah, I was kind of torn on this as well and I didn't want to maintain the three arrays, but I like the idea of having a getter or a function that does this concatenation.